ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -707,6 +707,129 @@ <h3 class="title">
       </div>
     </mat-tab>
 
+    <mat-tab>
+      <ng-template mat-tab-label>
+        <span>CHECK CONSTRAINTS</span>
+      </ng-template>
+      <div class="cc-tab-container">
+        <table mat-table [dataSource]="ccDataSource">
+          <ng-container matColumnDef="srcDatabase">
+            <th mat-header-cell class="db_name" *matHeaderCellDef [attr.colspan]="3">{{ srcDbName }}</th>
+          </ng-container>
+
+          <ng-container matColumnDef="spDatabase">
+            <th mat-header-cell *matHeaderCellDef class="db_name" [attr.colspan]="4" >
+              <div class="spanner_edit-button">
+                <span>Spanner</span>
+                <div *ngIf="
+                    !isCcEditMode &&
+                    ccDataSource.length > 0 &&
+                    currentObject!.isSpannerNode &&
+                    !currentObject.isDeleted
+                  ">
+                  <button mat-stroked-button color="primary" (click)="toggleCcEdit()">
```

Comment:
If I am not wrong, the button will only be available if ccDataSource.length > 0. But its possible someone wants to add a check constraint where no constraint is already existing.

Reply:
resolve the comment